bouncer/shared/create_and_delete_multiple_orders.ts

```diff
@@ -20,34 +44,35 @@ export async function createAndDeleteMultipleOrders(numberOfOrders: number) {
   }[] = [];
   let i = 0;
   while (i < numberOfOrders) {
-    promises.push(limitOrder('Btc', 0.00000001, i, i));
+    promises.push(limitOrder('Btc', 0.00000001, i, i, '//LP_3'));
     orderToDelete.push({ Limit: { base_asset: 'BTC', quote_asset: 'USDC', side: 'sell', id: i } });
     i++;
   }
+  i = 0;
+  while (i < numberOfOrders) {
```

Comment:
Its misleading now that numberOfOrders does not include the range orders. Maybe rename it `numberOfLimitOrders`.